ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.16-nightly-x86-loaded-upgrade-from-4.15.yaml

```diff
@@ -42,16 +43,115 @@ tests:
       CHURN: "false"
       ENABLE_FORCE: "true"
       GC: "false"
+      ITERATION_MULTIPLIER_ENV: "6"
```

Comment:
@vishnuchalla Is there some issue we met that we want to use 6 instead of default 9? 
I noticed in different tests ITERATION_MULTIPLIER_ENV is set to different values. For example payload-control-plane-6nodes: 15. 
Is there a guide line for ITERATION_MULTIPLIER_ENV's config?

Reply:
We should use 9 here, as we standardized on 9 as a safe value for all scales, up to 252 nodes.

Reply:
And since I am the one who added `payload-control-plane-6nodes` with 15 and I don't recall exactly why: I reviewed the data from that story and p99 latency was still 13s at 15, so I went with that.

I mentioned 9 being a "safe value", that is: "conservatively low to keep latencies reliably at 13s", which is our starting point. Eventually, I hope to increase the value and I think 15 in the payload test is a start on that.

Reply:
For 24 nodes I see 6 iterations in 4.14 as well. And for rest of the scales I see 9. Looks this is a common pattern that we follow. Do we want to change it to 9 for all the scales?

Reply:
@afcollins any thoughts around this?

Reply:
Hmm, lost track of this one..

the value of 6 was introduced in 7ee83dcfd64d6b608df51038020d7f376d1441ca . I feel let's not propagate that and use the standard value of 9 here and ignore payload tests for now.

Reply:
@afcollins @vishnuchalla  let's stick to 9 as `ITERATION_MULTIPLIER`, we have tested this enough times recently at very large scale like 252/501 with 9 as the magic number